### ISSUE TITLE
False error hint for angular 2 directives

### ISSUE THREAD

**Issue by engelux** [AtomLinter/linter-htmlhint#132]
The Angular 2 Doc says that ngModel have to be written in camelcase

![screenshot_6](https://cloud.githubusercontent.com/assets/6931748/17526693/f403e40a-5e2d-11e6-8956-a8c63903dc60.png)

**Comment by Arcanemagus** [comment]
Does `htmlhint` show the same message? That isn't valid _pure_ HTML, so it's likely you either need to tell `htmlhint` to ignore those errors, or disable the Angular 2 scope in the settings.

**Comment by engelux** [comment]
Ok, how i set htmlhint to ignore those errors?, thanks!

**Comment by Arcanemagus** [comment]
I would guess you create a ['.htmlhintrc'](https://github.com/yaniswang/HTMLHint/wiki/Usage#about-htmlhintrc) file containing something along the following:

```json
{
  "attr-lowercase": false
}
```

**Comment by engelux** [comment]
ohhh, ok, thanks! :)
